backend/app.py

```diff
@@ -0,0 +1,44 @@
+from flask import Flask, Response
+from flask_jwt_extended import JWTManager
+
+from .api.auth import auth
+from .api.documents import document_api
+from .api.invites import invite_api
+from .api.messages import message_api
+from .services.send_email import mail
+
+
+def create_flask_app() -> Flask:
+    application = Flask(__name__)
+
+    application.config["MONGO_URI"] = "mongodb://localhost:27017/myDatabase"
```

Comment:
I think all these items should be stored as env variables and exported before run app